foundation/foundation-upgrade/src/main/resources/migrations.json

```
@@ -104,6 +104,15 @@
             "includes": ["src/*/resources/apps/*.yaml"]
           }
         ]
+      },
+      {
+        "name": "fastapi-removal-yaml-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.version_specific.FastapiDependencyRemovalValuesMigration",
+        "fileSets": [
+          {
+            "includes": ["*-deploy/src/main/resources/apps/*/*.yaml"]
```

A: This should be relative to the deploy project and target values files specifically
```suggestion
            "includes": ["src/main/resources/apps/*/values*.yaml"]
```

---

Updated path!